src/Extend/Model.php

```diff
@@ -49,6 +51,7 @@ public function dateAttribute(string $attribute)
 
     /**
      * Add a default value for a given attribute, which can be an explicit value, or a closure.
+     * Unlike with some other extenders, it CANNOT be an invokable class.
```

Comment:
Technically, it can be an invokable class if you pass an object and not the class name as string.

It's the string-based invokable classes that are specifically not possible.